test/apis/email.test.ts

```diff
@@ -84,6 +85,64 @@ describe('Email', () => {
     );
   });
 
+  it('exposes a send method that accepts an attachment argument', async () => {
+    expect.assertions(1);
+    (axios as any).post.mockResolvedValue({});
+
+    let email = new Email({
+      baseUrl: BASE_URL,
+      username: USERNAME,
+      password: PASSWORD,
+    });
+    await email.send({
+      to: 'test@example.com',
+      from: 'Tests <testing@example.com>',
+      subject: 'Testing',
+      text: 'hello world',
+      attachment: [
+        {
+          data: Fs.readFileSync('./test/apis/email.test.ts'),
```

Comment:
🤔 very strange that this test is green, considering that the following test fails:
```
  it('throws if input is invalid', () => {
      expect(() => Validator.requiredString(Fs.readFileSync('./test/apis/email.test.ts'))).toThrowError();
  });
```
maybe some weird jest concurrent thing? 😅